ynput/OpenPype
# scene length setting referenced from DB record for 3dsMax

## Changelog Description
 Setting the timeline length based on DB record in 3dsMax Hosts
![image](https://user-images.githubusercontent.com/64118225/226571059-4ef7d1c7-ecc7-448c-a0d1-aa97f1159810.png)

## Additional info
N/A

## Testing notes:
1. launch 3dsMax via Openpype
2. create new scene
3. check the timeline
4. locate "Set Frame Range" in openpype menu
5. click it
6. check the frame range or the timeline to see if it is same with the setting shown in DB
